admin/c2cgeoportal_admin/views/layer_groups.py

```diff
@@ -1,12 +1,15 @@
+import deform
+import colander
 from functools import partial
 from pyramid.view import view_defaults
 from pyramid.view import view_config
 from c2cgeoform.schema import GeoFormSchemaNode
 from c2cgeoform.views.abstract_views import ListField
-from c2cgeoportal_commons.models.main import LayerGroup
 from c2cgeoportal_admin.schemas.treegroup import children_schema_node
 from c2cgeoportal_admin.schemas.metadata import metadatas_schema_node
 from c2cgeoportal_admin.views.treeitems import TreeItemViews
+from c2cgeoportal_admin.schemas.treegroup import base_deferred_parent_id_validator
```

Comment:
why do not import it on the same line as other imports from c2cgeoportal_admin.schemas.treegroup

Reply:
ok